src/main/java/org/folio/circulation/resources/CirculationRulesResource.java

```diff
@@ -89,19 +108,32 @@ private void get(RoutingContext routingContext) {
   @SuppressWarnings("squid:S2147")
   private void put(RoutingContext routingContext) {
     final Clients clients = Clients.create(new WebContext(routingContext), client);
-    CollectionResourceClient loansRulesClient = clients.circulationRulesStorage();
 
-    if (loansRulesClient == null) {
+    if (clients.circulationRulesStorage() == null) {
       internalError(routingContext.response(),
         "Cannot initialise client to storage interface");
       return;
     }
 
+    getExistingPolicyIds(clients)
+      .thenAccept(result -> proceedWithUpdate(result.value(), routingContext, clients));
+  }
+
+  private void proceedWithUpdate(
+    Map<String, Set<String>> existingPoliciesIds, RoutingContext routingContext, Clients clients) {
```

Comment:
Please can this be formatted with the standard separation, where the line is broken when 80 (or 100 is usually accepted) is reached